src/helpers/likeAndDislike.js

```diff
@@ -0,0 +1,24 @@
+export const fromDisLikeToLike = (likesUsers, dislikeUsers, id) => {
+  dislikeUsers.forEach((userId, index) => {
+    if (userId === id) {
+      dislikeUsers.splice(index, 1);
```

Comment:
This is causing O(n^2) and it is very bad, why not change it into O(n).

Reply:
> This is causing O(n^2) and it is very bad, why not change it into O(n).

@kagabof since my logic to that functionality has ended  you can give me an idea of how I can change into O(n)

Reply:
I have tried to use the paths I could think of but none is helping me